hardware_pydantic/junior/junior_base_devices.py

```diff
@@ -116,7 +114,6 @@ def action__dispense(
             destination_container: ChemicalContainer,
             dispenser_container: ChemicalContainer,
             amount: float,
-            dispense_speed: float = 5,
```

Comment:
1. If you have a model function to calculate time cost, instead of hardcoding, please add model parameters as arguments to action functions. This applies to all action functions.
2. The child function `action__dispense_sv` relies on `dispense_speed` to deal with cases where `powder_param_known` is `True`.